### PR TITLE
Update CHANGELOG format and parsing

### PR DESCRIPTION
**Pull request by JakeQuilty**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,74 +6,67 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Add support for `conjur.conf` in Windows system directory.
+
+## 0.3.3 - 2018-08-02
+### Changed
+- Update the tags on `PolicyResponse` so they match the JSON returned by the server.
+
+## 0.3.2 - 2018-07-19
+### Added
+- Use github.com/sirupsen/logrus for logging.
+- *When the log level for logrus is set to DebugLevel, show debug information, including:*
```
Comment (doodlesbykumbi) [inline CHANGELOG.md]: Nested lists are valid: I think that flattening them loses some useful nuance

```
- When the log level for logrus is set to DebugLevel, show debug information, including:
  - what configuration information is contained in each of the locations (e.g. the environment, config files, etc), as well as the final configuration
  -  HTTP request sent to, and the responses received from, the Conjur server
```

File: Jenkinsfile [hunk 2]
```diff
@@ -20,6 +20,14 @@ pipeline {
       }
     }
 
+    stage('Validate') {
```
Comment (doodlesbykumbi) [inline Jenkinsfile]: Please move this to earlier in the pipeline.

File: CHANGELOG.md (hunk 1 above)
Comment (JakeQuilty) [reply CHANGELOG.md]: Done

**Comment by doodlesbykumbi** [comment]
This PR is good. Please address https://github.com/cyberark/conjur-api-go/pull/59#discussion_r380724117 before we proceed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,74 +6,67 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+## [0.2.0] - 2018-01-08
+### Added
+- Adds support for structured error responses from the Conjur v5 server, using the struct `conjurapi.ConjurError`. This is a backwards incompatible change.
+- All API methods accept fully qualified object ids in v5 mode. This is a backwards compatible bug fix.
+- API methods which do not work in v4 mode return an appropriate error message. This is a backwards compatible bug fix.
+
+## 0.1.0 - 2017-11-21
+### Added
+- Initial version
```
Comment (diverdane) [inline CHANGELOG.md]: Can add a link for `0.3.2` and initial release:
```
[0.3.3]: https://github.com/cyberark/conjur-api-go/compare/v0.3.2...v0.3.3
[0.3.2]: https://github.com/cyberark/conjur-api-go/compare/v0.3.0...v0.3.2
[0.3.0]: https://github.com/cyberark/conjur-api-go/compare/v0.2.0...v0.3.0
[0.2.0]: https://github.com/cyberark/conjur-api-go/compare/v0.1.0...v0.2.0
[0.1.0]: https://github.com/cyberark/conjur-api-go/releases/tag/v0.1.0
```

**Pull request merged**